bedrock/firefox/templates/firefox/all.html

```diff
@@ -30,7 +30,7 @@
   sky firefox-all-{{ channel }} {% if channel == 'alpha' %}blueprint{% endif %}
 {% endblock %}
 
-{% block site_css %}
+{% block page_css %}
```

Comment:
The `firefox_all` bundle in `static_media.py` still references `sandstone-resp.less`, which results in duplicate CSS.